snap/implicit.go

```diff
@@ -43,6 +43,7 @@ var implicitClassicSlots = []string{
 	"opengl",
 	"network-manager",
 	"pulseaudio",
+	"modem-manager",
```

Comment:
Is modem manager really needed on classic or is this just something that works with network-manager? I'm trying to understand what snaps would legitimately access modem manager on classic.